Source/UserManagement/Events/MessageGenerated.cs

```diff
@@ -6,7 +6,7 @@ namespace Events
 {
     public class MessageGenerated : IEvent
     {
-        public Guid Id { get; set; }
+        public Guid Id { get; set; } //QUESTION: einari, michael: What does this represent? An EventSourceId or the StaffUserId of the datacollector?
```

Comment:
It should be the EventSourceId  which is the Staff User Id.  There's no need to embed the id of the event in the body of the event.  Personally, I think Id is ambiguous and should be avoided.  StaffUserId is a better name.